lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -47,7 +47,7 @@ def purchase(money, payment, options = {})
 
       def authorize(money, payment, options = {}, capture = false)
         post = build_auth_request(money, payment, options)
-        post[:processingInformation][:capture] = true if capture
+        post[:processingInformation][:capture] = true if capture && !options[:third_party_token]
```

Comment:
We usually designate this by checking if the payment method argument is a string
```suggestion
        post[:processingInformation][:capture] = true if capture && !payment.is_a?(String)
```